string tag
float32 floatValue
bool booleanValue